From 13b224d0b7442dd2e308be4631d53971c1bdf8bd Mon Sep 17 00:00:00 2001
From: Stephen Chu <schu@cloudera.com>
Date: Thu, 13 Nov 2014 10:21:27 -0800
Subject: [PATCH 175/596] CLOUDERA-BUILD. CDH-23226: Bump C5 Hadoop's jetty to
 6.1.26.cloudera.4

(cherry picked from commit d3c8b1c5c8dbba403570ae7c5fcd85d760534aa6)
(cherry picked from commit 79c51937798e5414008d0419ae92f75222fa122c)
---
 .../.eclipse.templates/.classpath                  |    4 ++--
 .../ivy/hadoop-core-pom-template.xml               |    4 ++--
 hadoop-mapreduce1-project/ivy/libraries.properties |    4 ++--
 hadoop-project/pom.xml                             |    6 +++---
 4 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/hadoop-mapreduce1-project/.eclipse.templates/.classpath b/hadoop-mapreduce1-project/.eclipse.templates/.classpath
index bdbda48..9e63109 100644
--- a/hadoop-mapreduce1-project/.eclipse.templates/.classpath
+++ b/hadoop-mapreduce1-project/.eclipse.templates/.classpath
@@ -33,8 +33,8 @@
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/log4j-1.2.15.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/mockito-all-1.8.2.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/oro-2.0.8.jar"/>
-	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-6.1.26.cloudera.3.jar"/>
-	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-util-6.1.26.cloudera.3.jar"/>
+	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-6.1.26.cloudera.4.jar"/>
+	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jetty-util-6.1.26.cloudera.4.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jackson-core-asl-1.5.2.jar"/>
 	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/jackson-mapper-asl-1.5.2.jar"/>
   	<classpathentry kind="lib" path="build/ivy/lib/Hadoop/common/servlet-api-2.5-6.1.14.jar"/>
diff --git a/hadoop-mapreduce1-project/ivy/hadoop-core-pom-template.xml b/hadoop-mapreduce1-project/ivy/hadoop-core-pom-template.xml
index 08e49ec..5abc497 100644
--- a/hadoop-mapreduce1-project/ivy/hadoop-core-pom-template.xml
+++ b/hadoop-mapreduce1-project/ivy/hadoop-core-pom-template.xml
@@ -61,12 +61,12 @@
     <dependency>
       <groupId>org.mortbay.jetty</groupId>
       <artifactId>jetty</artifactId>
-      <version>6.1.26.cloudera.3</version>
+      <version>6.1.26.cloudera.4</version>
     </dependency>
     <dependency>
       <groupId>org.mortbay.jetty</groupId>
       <artifactId>jetty-util</artifactId>
-      <version>6.1.26.cloudera.3</version>
+      <version>6.1.26.cloudera.4</version>
     </dependency>
     <dependency>
       <groupId>tomcat</groupId>
diff --git a/hadoop-mapreduce1-project/ivy/libraries.properties b/hadoop-mapreduce1-project/ivy/libraries.properties
index a8957cd..e2ba241 100644
--- a/hadoop-mapreduce1-project/ivy/libraries.properties
+++ b/hadoop-mapreduce1-project/ivy/libraries.properties
@@ -54,8 +54,8 @@ jdeb.version=0.8
 jsp.version=2.1
 jsp-api.version=5.5.12
 jets3t.version=0.7.1
-jetty.version=6.1.26.cloudera.3
-jetty-util.version=6.1.26.cloudera.3
+jetty.version=6.1.26.cloudera.4
+jetty-util.version=6.1.26.cloudera.4
 junit.version=4.8.1
 jdiff.version=1.0.9
 
diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 11165c7..5b5d10f 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -414,7 +414,7 @@
       <dependency>
         <groupId>org.mortbay.jetty</groupId>
         <artifactId>jetty</artifactId>
-        <version>6.1.26.cloudera.3</version>
+        <version>6.1.26.cloudera.4</version>
         <exclusions>
           <exclusion>
             <groupId>org.mortbay.jetty</groupId>
@@ -425,7 +425,7 @@
       <dependency>
         <groupId>org.mortbay.jetty</groupId>
         <artifactId>jetty-util</artifactId>
-        <version>6.1.26.cloudera.3</version>
+        <version>6.1.26.cloudera.4</version>
       </dependency>
       <dependency>
         <groupId>org.apache.tomcat.embed</groupId>
@@ -538,7 +538,7 @@
       <dependency>
         <groupId>org.mortbay.jetty</groupId>
         <artifactId>jetty-servlet-tester</artifactId>
-        <version>6.1.26.cloudera.3</version>
+        <version>6.1.26.cloudera.4</version>
       </dependency>
       <dependency>
         <groupId>tomcat</groupId>
-- 
1.7.9.5

